src/lightning_app/utilities/frontend.py

```diff
@@ -14,10 +14,15 @@ class AppInfo:
     meta_tags: Optional[List[str]] = None
 
 
-def update_index_file_with_info(ui_root: str, info: AppInfo = None) -> None:
+def update_index_file_with_info_and_root_path(
```

Comment:
perhaps just `update_index_file`?

Reply:
the arguments says what's being used for updating it

Reply:
good point!

Reply:
updated!